sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12.processing.service.ts

```diff
@@ -216,7 +216,7 @@ export class IER12ProcessingService {
         assessmentId: pendingAssessment.id,
         disbursementId: disbursement.id,
         applicationNumber: application.applicationNumber,
-        institutionStudentNumber: application.data.studentNumber,
+        institutionStudentNumber: application.studentNumber,
```

Comment:
isn't the `studentNumber` an optional field in the form?

Reply:
It is, and we not changing it. The `application.data.studentNumber` is saved to `application.studentNumber` when the application is submitted. This change is just to get the value from the DB instead dynamic data.

Reply:
Yup I got that, My comment was should be then make the `institutionStudentNumber` optional. and it is already optional, so resolving my comment